tests/e2e/files/rulebooks/operators/test_not_operator.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Test not (negation) operator
+  hosts: all
+  sources:
+    - generic:
+        event_delay: 1
```

Comment:
We should not need the delay. If you remove it, it doesn't work. I think it's a bug.

Reply:
It's not a bug, the source plugin ends before. But we can gain one second replacing the delay by `shutdown_after: 1`

Reply:
Using `shutdown_after` still allows for the possibility that the plugin ends before the run_playbook action has completed. Out of 100 tests it failed 14 times due to this. Using `event_delay` there were no failures and the difference in delay was only a matter of a few ms, so will leave it as-is, if you agree.

Reply:
ok